ui/app/models/task-group.js

```diff
@@ -14,10 +14,10 @@ export default Fragment.extend({
 
   tasks: fragmentArray('task'),
 
+  services: fragmentArray('service'),
+
   drivers: computed('tasks.@each.driver', function() {
-    return this.tasks
-      .mapBy('driver')
-      .uniq();
+    return this.tasks.mapBy('driver').uniq();
```

Comment:
Same `get` or not to `get` question here, I saw this second one and thought 'oh they must be migrating', but then I saw elsewhere in this changeset you are using `get` - so I dunno - just a flag here incase you should be doing one or the other.

Reply:
This diff is sadly the result of a Git autoformatting hook 😐

Reply:
Ah yeah 😆 whitespace! Sorry, I saw the get-less thing here and missed it was just a whitespace change! Please ignore!